src/app/components/popup-choose-the-time/popup-choose-the-time.module.ts

```diff
@@ -0,0 +1,32 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { PopupChooseTheTimeComponent } from './popup-choose-the-time.component';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { MatSelectModule } from '@angular/material/select';
+import { ModalWindowModule } from '../modal-window/modal-window.module';
+import { MatDatepickerModule } from '@angular/material/datepicker';
+import { MatInputModule } from '@angular/material/input';
+import { OwlDateTimeModule, OwlNativeDateTimeModule } from 'ng-pick-datetime';
+import { BrowserModule } from '@angular/platform-browser';
+import { TimeModule } from './time/time.module';
+
```

Comment:
do you need in all these modules? verify please

Reply:
removed unnecessary